Victor-Ugwueze/MyDiary
# #162361682 return user object after authentication

#### What does this PR do?
- return user object after authentication

#### Description of Task to be completed?
- return user object after login
- return user object after signup

#### How should this be manually tested?
- start the app
- login by providing the required fields
- you should get a user a token and user details

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#162361682](https://www.pivotaltracker.com/story/show/162361682, "return user object after authentication")

#### Screenshots (if appropriate)
NA

#### Questions:
